benchmark/sirun/get-results.js

```diff
@@ -0,0 +1,73 @@
+'use strict'
```

Comment:
What is the purpose of this file? Is it something sirun should do?

Reply:
Wasn't there supposed to be support for different target environments like CircleCI to generate the YAML etc? Also, I still don't understand what is the purpose of this file.

Reply:
> Wasn't there supposed to be support for different target environments like CircleCI to generate the YAML etc?

Yeah. But that was only really discussed in terms of defining tests, not grabbing their results.

> Also, I still don't understand what is the purpose of this file.

The file exists as a quick-and-easy tool to get all the sirun test results for a given commit from CircleCI artifacts. If that's objectionable, I can remove it from the PR.